superset/assets/src/visualizations/nvd3_vis.js

```diff
@@ -869,7 +869,9 @@ export default function nvd3Vis(slice, payload) {
         // Display styles for Time Series Annotations
         d3.selectAll('.slice_container .nv-timeseries-annotation-layer.showMarkerstrue .nv-point')
           .style('stroke-opacity', 1)
-          .style('fill-opacity', 1);
+          .style('fill-opacity', 1)
+          .style('stroke-width', (vizType === 'line' && fd.annotation_layers && fd.annotation_layers.length > 0)?
+            fd.annotation_layers[0].markerWidth: 1);
```

Comment:
We can have multiple annotation layer why we set this only on first?

Reply:
this is just a safe check with vizType `line`, as the marker option will only be available with TimeSeries viz

Reply:
Fixed